pages/cloud/using.md

```diff
@@ -1,5 +1,284 @@
-# Using LakeEffect
+## Using the LakeEffect Research Cloud
 
-!!! todo
+### Managing your Project with the Cloud Dashboard
 
-   write me
+Once you've requested and been granted an account on the Lake Effect cloud, you will be able to login in to the web console.
+Your username and password for Lake Effect are the same as your CCR username and password.  However, you need additional 
+privileges to be allowed to login to the cloud.  You will be informed when your cloud account is ready.
+
+[The Lake Effect Dashboard (Horizon)](https://dashboard.cloud.ccr.buffalo.edu) - simplifies management of your project's cloud resources and provides information on your group's usage.
+
+![](../images/cloud/openid.png)
+
+
+You will need to authenticate to CCR's **Identity Management System** with your CCR username and password+otp
+For more information about logging in with two-factor authentication enabled click [here](../2fa.md).
+
+![](../images/cloud/ccridm.png)
```

Comment:
I would recommend removing this screenshot as the login process will be changing shortly and looks different.  Users will be prompted to enter their username, then taken to the next page where they enter their password and, in a separate box, OTP code.

Reply:
stuck in a new screenshot.